certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -5392,19 +5486,19 @@
           "200": {
             "description": "Success",
             "schema": {
-              "$ref": "#/definitions/DtoUtilityBaseResponseBool"
+              "$ref": "#/definitions/DtoUtilityBaseResponseString"
```

Comment:
@Encodian The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed.
Can you please confirm that no users will be affected by this change?

Reply:
@Amjed-Ayoub - The change to update the schema of the /api/v1/Mgmt/GetOperationStatusUtilityString path is not a breaking change. This can be accepted as we are handling the update and this will not cause an issue with any existing flows.

Reply:
@Encodian thank you so much for the clarification.